ui-ngx/src/app/modules/home/home.component.ts

```diff
@@ -144,9 +146,20 @@ export class HomeComponent extends PageComponent implements AfterViewInit, OnIni
 
   private updateActiveComponent(activeComponent: any) {
     this.showSearch = false;
+    this.hideLoadingBar = false;
     this.textSearch.reset('', {emitEvent: false});
     this.activeComponent = activeComponent;
-    this.hideLoadingBar = activeComponent && activeComponent instanceof RouterTabsComponent;
+    let showLoadingBar: boolean;
+    if (isDefined(this.activeComponent.activatedRoute?.data?.value?.showLoadingBar)) {
+      showLoadingBar = this.activeComponent.activatedRoute?.data?.value?.showLoadingBar;
```

Comment:
A good change from  `this.activeComponent.activatedRoute?.data?.value?` to  `this.activeComponent.activatedRoute.snapshot.data`